xds/googledirectpath/googlec2p_test.go

```diff
@@ -126,6 +128,26 @@ func expectedNodeJSON(ipv6Capable bool) []byte {
 	}`)
 }
 
+// verifyXDSClientBootstrapConfig checks that an xDS client with the given name
+// exists in the pool and that its bootstrap config matches the expected config.
+func verifyXDSClientBootstrapConfig(t *testing.T, pool *xdsclient.Pool, name string, wantConfig *bootstrap.Config) xdsclient.XDSClient {
```

Comment:
Let's not return the xds client from this function. The only place the returned value is used in the last test. See comment there as well.